meilisearch/models/task.py

```diff
@@ -13,18 +13,18 @@ class Task(CamelBase):
     details: Dict[str, Any]
     error: Union[Dict[str, Any], None]
     canceled_by: Union[int, None]
-    duration: str
-    enqueued_at: str
-    started_at: str
-    finished_at: str
+    duration: Union[datetime, str]
+    enqueued_at: Union[datetime, str]
+    started_at: Union[datetime, str]
+    finished_at: Union[datetime, str]
```

Comment:
```suggestion
    duration: Optional[str]
    enqueued_at: datetime
    started_at: Optional[datetime]
    finished_at: Optional[datetime]
```

Reply:
`duration`, `started_at` and `finished_at` can be `None` but not string?

Reply:
Also `enqueued_at` has to be `datetime` ?

Reply:
All 3 can be `None`. `started_at` and `finished_at` will be a `datetime` if they aren't `None` and `duration` will be a `str`.

Reply:
yes, `enqueued_at` will always be a `datetime`

Reply:
Also, can't `duration` be `datetime`?

Reply:
no, the format it comes back in requires a string. something like `5 ms` if I remember correctly.